packages/edit-site/src/components/add-new-pattern/index.js

```diff
@@ -95,20 +101,21 @@ export default function AddNewPattern() {
 		onClick: () => {
 			patternUploadInputRef.current.click();
 		},
-		title: __( 'Import pattern from JSON' ),
+		title: __( 'Import from JSON' ),
```

Comment:
Is this change required, or should we keep specifying that we can only import patterns (not template parts)

Reply:
That's a good question. Designs suggested to remove this, but would like some confirmation from @jameskoster or @jasmussen